challenges/functions.js

```diff
@@ -6,14 +6,54 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+        const persons = [
```

Comment:
You were supposed to make a `consume` function that accepts two arguments and callback.